giskard/scanner/robustness/text_transformations.py

```diff
@@ -283,7 +283,11 @@ def _load_dictionaries(self):
 
     def make_perturbation(self, row):
         # Replace numbers with words
-        return self._regex.sub(lambda x: num2words(x.group(), lang=row["language__gsk__meta"]), row[self.column])
+        value = row[self.column]
+        lang = row["language__gsk__meta"]
+        if pd.isna(value) or pd.isna(lang):
+            return value
```

Comment:
if `not pd.isna(value) and pd.isna(lang)` I would still apply `num2words` without specifying `lang` (it would default to english).

Reply:
I changed the defaulting to en